### PR TITLE
(Breaking) Move away from using the forms registry

### PR DESCRIPTION
**Pull request by samuelmale**
Ticket: https://ohri.atlassian.net/browse/OHRI-1411

### CODE REVIEW

**Comment by samuelmale** [comment]
@larslemos @pirupius  any idea as to why the build is failing?

File: package.json [hunk 1]
```diff
@@ -96,7 +96,8 @@
   },
   "resolutions": {
     "@types/react": "^18.0.14",
-    "@types/react-dom": "^18.0.5"
+    "@types/react-dom": "^18.0.5",
+    "@openmrs/openmrs-form-engine-lib": "portal:/Users/sam/openmrs-form-engine-lib"
```
Comment (pirupius) [inline package.json:100]: u left this in. It might be the cause of the CI failure

File: packages/esm-commons-lib/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,326 @@
+{
```
Comment (pirupius) [inline packages/esm-commons-lib/package-lock.json:1]: This seems to be a mistake. We're using yarn for package management and we already have a `yarn.lock`

**Review by pirupius — CHANGES_REQUESTED** [review]

File: packages/esm-commons-lib/package-lock.json (hunk 2 above)
Comment (samuelmale) [reply packages/esm-commons-lib/package-lock.json]: Nice catch

File: packages/esm-commons-lib/src/utils/ohri-forms-commons.ts [hunk 3]
```diff
@@ -36,6 +43,8 @@ export const launchFormInEditMode = (
     moduleName,
   });
 };
+
+// @deprecated
```
[the remark this answers is not in the document]
Comment (samuelmale) [reply packages/esm-commons-lib/src/utils/ohri-forms-commons.ts:47]: Nope. The form registry pattern is dropped by this PR. However, these functions can still launch a form in a workspace

File: packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/hiv-exposed-infant.component.tsx [hunk 4]
```diff
@@ -279,11 +280,12 @@ const HivExposedInfant: React.FC<{
       <div style={{ padding: '1rem' }}>
         <EncounterList
           patientUuid={patientUuid}
-          encounterUuid={infantPostnatalEncounterType}
+          encounterType={infantPostnatalEncounterType}
           columns={columnsChildPreviousVisit}
           description={previousVisitsTitle}
           headerTitle={previousVisitsTitle}
-          form={{ package: 'child_health', name: 'infant_postnatal' }}
+          // TODO: replace with form name as configured in the backend.
```
[the remark this answers is not in the document]
Comment (samuelmale) [reply packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/hiv-exposed-infant.component.tsx:287]: This PR is done, added the TODOs as result of this thread: https://ucsf-gsi.slack.com/archives/C01GQG3GKDG/p1685974198518519

**Review by pirupius — DISMISSED** [review]

**Pull request merged**